src/frontend/src/utils/urls.ts

```diff
@@ -52,9 +52,20 @@ const workPackages = (queryParams?: { [field: string]: string }) => {
     .map((param) => `${param}=${queryParams[param]}`)
     .join('&')}`;
 };
+
+const workPackageTemplates = (queryParams?: { [field: string]: string }) => {
+  const url = `${API_URL}/template`;
+  if (!queryParams) return url;
+  return `${url}?${Object.keys(queryParams)
+    .map((param) => `${param}=${queryParams[param]}`)
+    .join('&')}`;
+};
+
```

Comment:
No need for this, i think you can just hardcode the URL

Reply:
bruh